src/main/resources/assets/presencefootsteps/lang/de_de.json

```diff
@@ -0,0 +1,67 @@
+{
+	"key.presencefootsteps.settings": "Presence Footsteps",
+	"pf.default_sounds.name": "Standard Sound Packet",
+	"menu.pf.title": "Presence Footsteps Optionen",
+	"menu.pf.multiplayer.true": "Einzel + Mehrspieler",
+	"menu.pf.multiplayer.false": "Nur Einzelspieler",
+	"menu.pf.global.all": "Entitäten: Alle",
+	"menu.pf.global.players_and_hostiles": "Entitäten: Spieler + Feindliches",
+	"menu.pf.global.players_only": "Entitäten: Nur Spieler",
+	"menu.pf.volume.min": "Gesamtlautstärke: AUS",
+	"menu.pf.volume": "Gesamtlautstärke: %d%%",
+	"menu.pf.volume.tooltip": "Die Gesamtlautstärke aller Presence Footsteps Geräusche. Setze auf 0 wenn du die Mod deaktivieren möchtest.",
+	"menu.pf.volume.hostile_entities": "Feindliche Entitäten: %d%%",
+	"menu.pf.volume.hostile_entities.tooltip": "Ändert die Lautstärle von feindlichen Laufgeräuschen.",
+	"menu.pf.volume.passive_entities": "Neutrale Entitäten: %d%%",
+	"menu.pf.volume.passive_entities.tooltip": "Ändert die Lautstärke von neutralen Laufgeräuschen.",
+	"menu.pf.volume.player": "Eigener Spieler: %d%%",
+	"menu.pf.volume.player.tooltip": "Ändert die Lautstärke des eigenen Spielers.",
+	"menu.pf.volume.other_players": "Andere Spieler: %d%%",
+	"menu.pf.volume.other_players.tooltip": "Ändert die Lautstärke der Laufgeräuschen anderer Spieler im Multiplayer.",
+	"menu.pf.volume.running": "Geschwindigkeitsanpassung: %d%%",
+	"menu.pf.volume.running.tooltip": "Wie stark die Lautstärke der Laufgeräusche beim Rennen variieren kann.\n\nDieser Wert wird zur Gesamtlautstärke HINZUGEFÜGT und skaliert mit Ihrer Geschwindigkeit.\n\nEs empfiehlt sich bei kleinen Werten zu starten und langsam anzupassen. Größere Werte verstärken den Effekt.",
```

Comment:
„Ihrer Geschwindigkeit“ → „deiner Geschwindigkeit“
(formal address form unusual for Minecraft environment)

„Es empfiehlt sich**,** bei kleineren Werten“